### ISSUE TITLE
ops failing to run on ubuntu server

### ISSUE THREAD

**Issue by Lost-in-Space** [nickthecook/ops#59]
Ops Failing with stack trace for both custom, or built-in actions (tested with `up`, `help` and noop custom action)

ops_versions: 1.3.0
os: Ubuntu 18.04.1 (#42~18.04.1-Ubuntu)
kernel: Linux 5.4.0 (5.4.0-1040-azure)
ruby version: 2.5.1p57

Error Stack Trace
```
Traceback (most recent call last):
	16: from /usr/local/bin/ops:23:in `<main>'
	15: from /usr/local/bin/ops:23:in `load'
	14: from /var/lib/gems/2.5.0/gems/ops_team-1.3.0/bin/ops:29:in `<top (required)>'
	13: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	12: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	11: from /var/lib/gems/2.5.0/gems/ops_team-1.3.0/lib/ops.rb:13:in `<top (required)>'
	10: from /var/lib/gems/2.5.0/gems/require_all-1.5.0/lib/require_all.rb:152:in `require_rel'
	 9: from /var/lib/gems/2.5.0/gems/require_all-1.5.0/lib/require_all.rb:152:in `each'
	 8: from /var/lib/gems/2.5.0/gems/require_all-1.5.0/lib/require_all.rb:153:in `block in require_rel'
	 7: from /var/lib/gems/2.5.0/gems/require_all-1.5.0/lib/require_all.rb:106:in `require_all'
	 6: from /var/lib/gems/2.5.0/gems/require_all-1.5.0/lib/require_all.rb:106:in `each'
	 5: from /var/lib/gems/2.5.0/gems/require_all-1.5.0/lib/require_all.rb:108:in `block in require_all'
	 4: from /var/lib/gems/2.5.0/gems/require_all-1.5.0/lib/require_all.rb:243:in `__require'
	 3: from /var/lib/gems/2.5.0/gems/require_all-1.5.0/lib/require_all.rb:243:in `require'
	 2: from /var/lib/gems/2.5.0/gems/ops_team-1.3.0/lib/builtins/countdown.rb:3:in `<top (required)>'
	 1: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- concurrent (LoadError)
```

**Comment by Lost-in-Space** [comment]
same system reverting to version 1.0.0 of ops team was successful

**Comment by Lost-in-Space** [comment]
ops_team version `1.2.4` also works as expected. Issue seems to have been introduced in latest version, (`1.3.0`)

**Comment by nickthecook** [comment]
Should be fixed in 1.3.1, which is now available.
